docs/rules/max-classes-per-file.md

```diff
@@ -28,8 +28,12 @@ class Foo {}
 
 ## Options
 
-This rule has a numeric option (defaulted to 1) to specify the
-maximum number of classes.
+This rule may take in either an object or a number.
+
+If the option is an object, it may contain one or both of:
+
+-   `ignoreExpressions`: a boolean option (defaulted to `false`) to ignore class expressions.
+-   `maximum`: a numeric option (defaulted to 1) to specify the maximum number of classes.
```

Comment:
We generally use “max” for this purpose in other rules, so good to be consistent. 

```suggestion
-   `max`: a numeric option (defaulted to 1) to specify the maximum number of classes.
```